styles/main.css

```diff
@@ -244,8 +250,8 @@ header nav .burger {
     .paperclip {
         display: none;
     }
-    .mugshot {
-        float: unset;
+    main>.bio>.group>.mugshot {
+        float: none;

```

Comment:
You need your selector to be at least as specific as the selector which it is supposed to override
Use `none` instead of `unset` to remove the float